config/environments/deployed_development.rb

```diff
@@ -57,6 +57,10 @@
   config.gias_api_user = ENV["GIAS_API_USER"]
   config.gias_api_password = Rails.application.credentials.GIAS_API_PASSWORD
 
+  config.dqt_client_api_key = Rails.application.credentials.DQT_CLIENT_API_KEY
+  config.dqt_client_host = Rails.application.credentials.DQT_CLIENT_HOST
+  config.dqt_client_params = Rails.application.credentials.DQT_CLIENT_PARAMS
```

Comment:
Are we sure all three of them belong in credentials? I would think only the key belongs there, everything else can live in unencrypted terraform files (`terraform/workspace-variables/review_app_env.yml` for example)

Reply:
Client params is definitely secret, contains a key. This is the pattern we followed for GIAS. It just seems polite not to leak the API endpoint, it's not public anywhere currently I don't think